pkg/xmtp/listener.go

```diff
@@ -75,6 +79,9 @@ func (l *Listener) startMessageListener() {
 			l.logger.Error("error connecting to stream", zap.Error(err))
```

Comment:
I would have expected the connection errors for being blocked by the WAF to be caught here, but I have 35mm log messages in DataDog that says that it only caught exceptions below.